sphinx/source/docs/user_guide/geo.rst

```diff
@@ -67,11 +67,11 @@ Bokeh's ``GeoJSONDataSource`` can be used almost seamlessly in place of Bokeh's
     :source-position: above
 
 .. warning::
-    It is important to note that behind the scenes, Bokeh converts the
-    GeoJSON coordinates into columns called `x` and `y` or `xs` and `ys`)
-    (depending on whether the features are Points, Lines, MultiLines, Polygons
+    It is important to note that, behind the scenes, Bokeh converts the
```

Comment:
I really dislike the comma after "that", maybe just delete "It is important to note that" altogether.

Reply:
That part can definitely be left out, I'll delete it with the next commit.